modules/core/24-host/v2/packet_keys.go

```diff
@@ -0,0 +1,24 @@
+package v2
+
+import "fmt"
+
+// PacketReceiptKey returns the store key of under which a packet
+// receipt is stored
+func PacketReceiptKey(sourceID string, sequence uint64) []byte {
+	return []byte(fmt.Sprintf("receipts/channels/%s/sequences/%d", sourceID, sequence))
```

Comment:
we can introduce utility functions if needed, but I didn't think it was necessary here